ginivision/src/main/java/net/gini/android/vision/internal/camera/api/CameraController.java

```diff
@@ -336,24 +337,24 @@ private void configureCamera(Activity activity) {
 
         Camera.Parameters params = mCamera.getParameters();
 
-        List<Camera.Size> previewSizes = params.getSupportedPreviewSizes();
-        Size previewSize = getLargestFourThreeRatioSize(previewSizes);
-        if (previewSize != null) {
-            mPreviewSize = previewSize;
-            params.setPreviewSize(mPreviewSize.width, mPreviewSize.height);
-            LOG.debug("Preview size ({}, {})", mPreviewSize.width, mPreviewSize.height);
-        } else {
-            LOG.warn("No 4:3 preview size found");
-        }
-
         List<Camera.Size> pictureSizes = params.getSupportedPictureSizes();
```

Comment:
IMO, the `configureCamera` method is starting to get overburdened. It does several things:
* size selection
* Focus
* Flash
* Settings configuration

I'd extract the size selection in a separate method, so at least it doesn't expose the selection algorithm.